File: src/main/java/me/aragot/hglmoderation/tools/StringUtils.java
package me.aragot.hglmoderation.tools;

public class StringUtils {

    public static String capitalize(String input){
        return input.substring(0, 1).toUpperCase() + input.substring(1);
    }

    public static String prettyEnum(Enum rawEnum){
        String [] splitEnumName = rawEnum.name().toLowerCase().split("_");
        StringBuilder builder = new StringBuilder();

        for(String part : splitEnumName)
            builder.append(capitalize(part)).append(" ");

        builder.replace(builder.length() - 1, builder.length(), "");
        return builder.toString();
    }
}

File: src/main/java/me/aragot/hglmoderation/service/StringUtils.kt
package me.aragot.hglmoderation.service

import java.util.*

class StringUtils {
    companion object {
        private fun capitalize(input: String): String {
            return input.substring(0, 1).uppercase(Locale.getDefault()) + input.substring(1)
        }

        fun prettyEnum(rawEnum: Enum<*>): String {
            val splitEnumName = rawEnum.name.lowercase(Locale.getDefault()).split("_".toRegex()).toTypedArray()
            val builder = StringBuilder()
            for (part in splitEnumName) builder.append(capitalize(part)).append(" ")
            builder.replace(builder.length - 1, builder.length, "")
            return builder.toString()
        }
    }
}